### PR TITLE
Change quick start guide

### PR DESCRIPTION
**Pull request by subatoi**
This in effect simplifies and expands explanations of base URLs and endpoints, beyond simply extending the quick start guide

### CODE REVIEW

File: source/documentation/getting_started/quick_start.md
Comment (danmitchell-) [inline source/documentation/getting_started/quick_start.md]: Not sure if `Registers collection page` is the correct term now. Might be worth checking with Connor - https://registers-trial.service.gov.uk/registers

File: source/documentation/getting_started/quick_start.md
Comment (danmitchell-) [inline source/documentation/getting_started/quick_start.md]: Could `/field-value` be better described as `/key` i.e. not to confuse with `fields` 🤔

**Comment by danmitchell-** [comment]
This is a fantastic improvement. Will let the powers that be have a final say ;-)

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [inline source/documentation/getting_started/quick_start.md]: Probably need to find a way to avoid saying "that share a field value" so @danmitchell-'s idea of renaming it "key" could be useful here.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [inline source/documentation/getting_started/quick_start.md]: Should we be describing this as a "RESTful" or a "REST" API? Information I've read online and conversations I've had are conflicting.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [inline source/documentation/getting_started/quick_start.md]: Would it actually be more helpful to have a three-column table with "Method" "HTTP request" and "Description"?

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [reply source/documentation/getting_started/quick_start.md]: I agree with `field-value` not being clear enough.  I think though, it is a job that needs a review across the whole documentation and probably out of scope for this PR.

**Comment by bahmady** [comment]
Thanks @danmitchell- —I've added quite a few comments so between mine and yours there's now a fair bit to review in this PR.

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [reply source/documentation/getting_started/quick_start.md]: if it helpls users frame the type of API, I don't see why not.

**Comment by arnau** [comment]
After a conversation with @bahmady, I don't see how the table in (2) helps a user get a sense of what can be done with the API.  I think a list of operations (perhaps like the description column) seems to give enough information.  Then, each item can link to the section explaining the technical details: HTTP method, path, parameters, options to consider, etc.

What do you think?

**Comment by nacnudus** [comment]
Maybe I'm weird but I prefer to read the second column of the table in (2), rather than the column with the URL templates.  I would also click on links to further explanation of "item", "entry" and "record" if they existed, because this is the point at which I realise that I don't know which one I want.

**Comment by bahmady** [comment]
I'm inclined to agree with @arnau ... I'll open up a new commit now and ask for feedback.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: I did check with Conor but he wasn't entirely sure going forwards what we'll be calling it, so latest PR just changes it to "here".

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: Yes, a bit careless of me re. "key", sorry. Latest commit for this specific PR addresses the backticks around "field-value".

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: Is the "are in beta" helping clarify that it is the same as "ready to use"? Or it's something to remove?

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: I'm probably picky here but, is the "here" link a good practice in terms of accessibility? Back in the day links had to aim to be meaningful without contextual information so in this case the link would've been "list of register base URLs"

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: 👌

File: source/documentation/getting_started/quick_start.md
Comment (danmitchell-) [reply source/documentation/getting_started/quick_start.md]: Aside from the content the linked page doesn't show "base URL's"

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: Technically this is the "Register ID" not the "Register name". Once the work done by @michaelabenyohai gets merged it will be more obvious.

E.g. 
Name: Local Authorities in England
ID: local-authority-eng

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: it should be `GET /record/BIR` to fully express that the path is absolute.

File: source/documentation/getting_started/quick_start.md
Comment (arnau) [inline source/documentation/getting_started/quick_start.md]: Do we have elsewhere the full table with format, suffix and media type?

format|suffix|media type
------ | ---- | -----------
JSON|.json|application/json
YAML|.yaml|not registered, what are we accepting here?
CSV|.csv|text/csv
TSV|.tsv|text/tsv
Turtle|.ttl|text/ttl

Comment (arnau) [reply source/documentation/getting_started/quick_start.md]: The spec says `text/yaml` http://openregister.github.io/specification/#yaml-representation

**Review by arnau — CHANGES_REQUESTED** [review]

File: source/documentation/getting_started/quick_start.md
Comment (nacnudus) [inline source/documentation/getting_started/quick_start.md]: Is this line still needed given the bullet points above?

File: source/documentation/getting_started/quick_start.md
Comment (nacnudus) [inline source/documentation/getting_started/quick_start.md]: Maybe this should link to [#get-records](#get-records) which is the only endpoint with an example of `page-size` and `page-index`.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: It's a holdover from old language I didn't write, so I would look to you to tell me if we should just remove it.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: @arnau Yes, this was just a case of me trying to cram in too much last night. You're right.  @danmitchell-'s correct as well. I'm inclined to think it would actually be clearer to get rid of this altogether and for me to write a new sentence which explains it better from context, without giving a list of base URLs.

Comment (arnau) [reply source/documentation/getting_started/quick_start.md]: 👍 to @danmitchell- suggestion

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: In this instance I can't think of a way to restructure this sentence since "beta" is such an important part of it:

> Registers which are in beta are reliable and ready for use in products and services.

So perhaps just delete this sentence?

Comment (arnau) [reply source/documentation/getting_started/quick_start.md]: I'm interested in other opinions, @michaelabenyohai @danmitchell-

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: The idea of putting this in was to reinforce the point by example, since previous user research indicated that people struggled with the descriptions surrounding "endpoint".

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: I've put together a new table which will be in the next commit 👍

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: After discussing this with @arnau we've rejigged this section and it'll be in the new commit. It will need revisiting in a future PR as well.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: Even if it's true I think it's not really necessary to put it in the API docs; it's the sort of information that really belongs in more of a frontend context IMO. I've removed the sentence in the latest commit but depending on further feedback I can revisit/reintroduce it.

File: source/documentation/getting_started/quick_start.md
Comment (nacnudus) [reply source/documentation/getting_started/quick_start.md]: It's quite far out of context now though.  This particular example is for the third of the endpoints above, and it's the only mention of `field-value`.  I think users would be better of following the links above to find out how to use a given endpoint.

**Review by arnau — APPROVED** [review]

**Comment by bahmady** [comment]
Would be great to get another review of the PR as it stands since it's quite an important change, but since it would be equally great to have this pushed in some state before Emma's UR, I will merge it before then.

File: source/documentation/getting_started/quick_start.md
Comment (bahmady) [reply source/documentation/getting_started/quick_start.md]: Yes, looking at it again after all the other changes, I agree with you. cc @arnau ?

**Review by arnau — APPROVED** [review]

**Review by dankmitchell — APPROVED** [review]
Nice

**Review by nacnudus — APPROVED** [review]

**Review by michaelabenyohai — APPROVED** [review]
👍

**Pull request merged**
